menu/additional_apps.sh

```diff
@@ -45,12 +45,12 @@ $CHECKLIST_GUIDE\n\n$RUN_LATER_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
 "Adminer" "(PostgreSQL GUI)" OFF \
 "Netdata" "(Real-time server monitoring in Web GUI)" OFF \
 "BPYTOP" "(Real-time server monitoring in CLI)" OFF \
-"Midnight Commander" "(CLI file manager)" OFF \
+"Midnight Commander" "(CLI file manager)" "$STARTUP_SWITCH" \
```

Comment:
I think Midnight Commander is a nice thing and makes more sense than Webmin, IMO.

Reply:
Most users need Webmin to even have any remote chance of managing their server because they don't have a clue about CLI. 

Midnight commander are for nerds like you and me, so no, I don't think it should be default.

Also, I never used MC, it's way better to learn the CLI commands instead. IMO :)